mitodl/ocw-hugo-themes
# remove manual, redundant inclusion of 4 deps bundled with nanogallery2

#### Pre-Flight checklist


#### What are the relevant tickets?
Toward #1033 

#### What's this PR do?
This PR removes 4 unnecessary dependencies from our bundles.

In short: bundling nanogallery2 with webpack is a bit of a pain
- originally, we were doing [this suggested workaround](https://github.com/nanostudio-org/nanogallery2/issues/98#issuecomment-391770383); but it results in duplicating 4 dependencies
- [this workaround](https://github.com/nanostudio-org/nanogallery2/issues/98#issuecomment-492121137) is better because it does not duplicate any dependencies.

#### How should this be manually tested?

Test image galleries with any course that supports them. For example, `2.672-spring-2009`.

1. Run `yarn start course`
2. Visit http://localhost:3000/pages/labs/ ... there will be links to many image galleries.
3. The image galleries won't actually work because the shortcode does not respect RESOURCE_BASE_URL. Temporarily edit the shortcode, `base-theme/layouts/shortcodes/image-gallery.html` so that `itemsBaseURL` points to `https://ocw.mit.edu/courses/2-672-project-laboratory-spring-2009/`:

    ```html
    <div id="{{ .Get "id" }}" class="image-gallery" data-nanogallery2='{ "itemsBaseURL": "https://ocw.mit.edu/courses/2-672-project-laboratory-spring-2009/" }'>
    {{ .Inner }}
    </div>
    ```
4. Check that the image galleries look the same as at https://ocw.mit.edu/courses/2-672-project-laboratory-spring-2009/


Some other courses with image galleries:
```
5.125a-fall-2005 ... under "Projects"
cms.608-spring-2008 under "image gallery"
```
